### PR TITLE
refactor: delete XXXContaining matchers

### PR DESCRIPTION
**Pull request by goosewobbler**
Closes https://github.com/webdriverio/expect-webdriverio/issues/1366

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -257,38 +216,6 @@ const myInput = await $('input')
...
-__Deprecated:__ please use `expect.stringContaining()` instead!
-
-##### Usage
-
-```js
-const myInput = await $('input')
-// 👎 deprecated
-await expect(myInput).toHaveAttributeContaining('class', 'form')
-// 👍 supported starting with v9
-await expect(myInput).toHaveAttribute('class', expect.stringContaining('form'))
```
Comment (christian-bromann) [inline docs/API.md:273]: Could we add this as example to the matchers so people see how they can assert against the full or partial text.

**Review by christian-bromann — CHANGES_REQUESTED** [review]

**Review by christian-bromann — APPROVED** [review]
👍 

I will keep this PR around until we are at the point where we release the first alpha v9 version.

**Comment by goosewobbler** [comment]
There are also some references to the removed functions in the main WDIO repo, in `eslint-plugin-wdio`.

**Comment by christian-bromann** [comment]
@goosewobbler good catch, we can remove them in `main`.

**Pull request merged**